parser/testdata/query/format/select_when_condition.sql

```diff
@@ -0,0 +1,7 @@
+-- Origin SQL:
+select case when false then 'hello' else 'world' end;
+
+-- Format SQL:
+
+SELECT 
+  CASE WHEN false THEN 'hello'ELSE 'world'END;
```

Comment:
It seems the space missing between 'hello' and ELSE.

Reply:
Yes, good catch.